README.md

```diff
@@ -115,11 +122,52 @@ async def show_current_power(data):
 @home.event("live_measurement")
 async def show_accumulated_cost(data):
   print(f"{data.accumulated_cost} {data.currency}")
-  
+
 def when_to_stop(data):
   return data.power < 1500
 
 # Start the live feed. This runs until data.power is less than 1500.
 # If a user agent was not defined earlier, this will be required here
-home.start_live_feed(user_agent = "UserAgent/0.0.1", exit_condition = when_to_stop) 
+home.start_live_feed(user_agent = "UserAgent/0.0.1", exit_condition = when_to_stop)
+```
+
+### Handling errors in a websocket connection
+
+```python
+import tibber
+
+from tibber.exceptions import ConnectionErrorList
+from tibber.exceptions import QueryErrorList
+from tibber.exceptions import MalformedQueryException
```

Comment:
These exceptions are not added yet. Either remove these or implement them before merging the PR.